api/http/logger.go

```diff
@@ -32,7 +32,7 @@ func routerLogger(logger logrus.FieldLogger) gin.HandlerFunc {
 		start := time.Now()
 		c.Next()
 		stop := time.Since(start)
-		latency := math.Ceil(float64(stop.Nanoseconds())) / 1000000.0
+		latency := math.Ceil(float64(stop.Nanoseconds())/1000000.0) / 1000.0
```

Comment:
multiplying is always cheaper than division, I would do `*0.001` instead of `/1000.0` otherwise lgtm

Reply:
Looks like golang is smart enough to convert divisions to multiplications for us